content/release-notes/2023-11-21-console.md

```diff
@@ -0,0 +1,10 @@
+---
+description: Read about the latest fixes and improvements in Neon Serverless Postgres.
+---
+
+### Fixes & improvements
+
+- Control Plane: Raised the PgBouncer `default_pool_size` limit, which defines the number of server connections allowed per user/database pair, from 16 to 64 to avoid running out of available server connections.
+- Integrations: Fixed an issue that permitted using a Neon project without a compute endpoint when adding the [Neon Vercel Integration](https://vercel.com/integrations/neon). 
+- UI: Updated the Prisma code snippet in the **Connection Details** widget. The revised code no longer includes the `connection_timeout` parameter or `shadowDatabaseUrl` variable. These are no longer required due to reduced Neon cold start times and support for database creation via SQL.
+- UI: Added an external link icon to the code snippet text box on the **Connection Details** widget to enable easy access to related documentation. 
```

Comment:
```suggestion
- UI: Added an external link icon to the code snippet text box on the **Connection Details** widget so that you can easily access related documentation. 
```